components/form/Probe.vue

```diff
@@ -191,7 +191,7 @@ export default {
             max="65535"
             :mode="mode"
             :label="t('probe.httpGet.port.label')"
-            :placeholder="t('probe.httpGet.port.placeholderDuex')"
+            :placeholder="t('probe.httpGet.port.placeholderDeux')"
```

Comment:
This is the corresponding change. But I still don't really know what it is

Reply:
Just a second placeholder, that's all.